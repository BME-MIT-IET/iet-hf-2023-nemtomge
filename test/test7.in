loadmap map.json 
start V1 
set Gencode LA1 Stun 30 30 
set Scientist V1 LA1 
give Active V1 Dementia 3 
touch V1
1
has Gencode V1 Stun
exit
