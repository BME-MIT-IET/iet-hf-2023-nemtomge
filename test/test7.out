Map from file map.json loaded
The game has been started. Players are: V1 
Genetic Code Stun aminoacid: 30 nucleotide: 30 has been placed on field LA1 
Scientist V1 set on field LA1 
Scientist V1 has been given active agent Dementia time: 3 
Choose one to pick: 
0 - cancel
1 - Stun(3)
Scientist V1 doesn't have gencode Stun
